kitsune/users/views.py

```diff
@@ -95,6 +104,13 @@ def login(request, template):
     form = handle_login(request, only_active=only_active)
 
     if request.user.is_authenticated():
+        if request.user.profile.is_fxa_migrated:
+            auth.logout(request)
```

Comment:
you can call the logout view from users/views.py which does the same thing and deleted the cookie.